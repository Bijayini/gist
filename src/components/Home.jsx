import React, { Fragment } from 'react';
import { Link } from 'react-router-dom';

import routePaths from '../lib/routePaths';

import BodyCopy from '../ui-components/BodyCopy';

import labels from './labels/Home.lables';

const Home = () => (
  <Fragment>
    <div className="home-page">
      <Link to={routePaths.GISTS}>
        <BodyCopy
          variant="extra-bold"
          xs="x-large"
          xm="x-large"
          xl="x-large"
          color="grey1"
        >
          {labels.welcome_message}
          &nbsp; &#x2192;
        </BodyCopy>
      </Link>
    </div>
    <style jsx="true">
      {`
        .home-page {
          text-align: center;
          margin-top: 200px;
        }
      `}
    </style>
  </Fragment>
);
export default Home;
