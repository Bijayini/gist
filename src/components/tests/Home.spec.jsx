import React from 'react';
import { shallow } from 'enzyme';

import Home from '../Home';

describe('<Home />', () => {
  let component;

  beforeEach(() => {
    component = shallow(<Home />);
  });

  it('should match with snapshot', () => {
    expect(component.debug()).toMatchSnapshot();
  });
});
