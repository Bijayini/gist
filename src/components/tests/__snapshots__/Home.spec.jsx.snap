// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> should match with snapshot 1`] = `
"<Fragment>
  <div className=\\"jsx-666615213 home-page\\">
    <Link to=\\"/gist-list\\" replace={false}>
      <BodyCopy variant=\\"extra-bold\\" xs=\\"x-large\\" xm=\\"x-large\\" xl=\\"x-large\\" color=\\"grey1\\" className=\\"\\" uppercase={false} styles={false}>
        Welcome to Gist List
          →
      </BodyCopy>
    </Link>
  </div>
  <JSXStyle id=\\"666615213\\">
    .home-page.jsx-666615213{text-align:center;margin-top:200px;}
  </JSXStyle>
</Fragment>"
`;
